layouts/shortcodes/unsafe-html.html

```diff
@@ -0,0 +1,16 @@
+{{/*
+
+Copyright 2020 Adobe
+All Rights Reserved.
+
+NOTICE: Adobe permits you to use, modify, and distribute this file in
+accordance with the terms of the Adobe license agreement accompanying
+it. If you have received this file from a source other than Adobe,
+then your use, modification, or distribution of it requires the prior
+written permission of Adobe.
+
+*/}}
+
+<!-- pass HTML as-is (raw) -->
+{{ .Inner }}
```

Comment:
should we add a small example of usage?

Reply:
Great idea @alexander-falca, an example of using `unsafe-html` shortcode to use a direct `<coral-*>` HTML Markup would be perfect.

Reply:
done